src/dnaio/__init__.py

```diff
@@ -69,7 +70,7 @@
 
 def open(
     file1: Union[str, PathLike, BinaryIO],
-    *,
+    *files: Union[str, PathLike, BinaryIO],
     file2: Optional[Union[str, PathLike, BinaryIO]] = None,
```

Comment:
technically `file1` should also be deprecated. This will make the interface much easier ->
```python3
dnaio.open("my.fastq", **kwargs) -> SingleEnd
dnaio.open("R1.fastq", "R2.fastq", **kwargs) -> PairedEnd
dnaio.open("R1.fastq", "R2.fastq", "R3.fastq", **kwargs) -> MultipleEnd
```
It will also become easier for us to use typing.overload and provide the correct typing information.

But in order to be backwards-compatible, file1 and file2 are kept intact.

Reply:
Hmm. That is a good idea actually. I will remake the interface so that `*files` is the primary way of entring files. File1 and file2 should be deprecated keyword arguments. I will change the logic to accomodate this. This will also make it much easier to write the docstring.

Reply:
I like that!